pkg/volume/fc/fc.go

```diff
@@ -126,17 +126,60 @@ func (plugin *fcPlugin) newMounterInternal(spec *volume.Spec, podUID types.UID,
 
 	return &fcDiskMounter{
 		fcDisk: &fcDisk{
-			podUID:  podUID,
+			podUID:  pod.UID,
+			volName: spec.Name(),
+			wwns:    fc.TargetWWNs,
+			lun:     lun,
+			wwids:   wwids,
+			manager: manager,
+			io:      &osIOHandler{},
+			plugin:  plugin},
+		fsType:     fc.FSType,
+		volumeMode: "",
```

Comment:
Why is volumeMode empty?

Reply:
Set v1.PersistentVolumeFilesystem to volumeMode .